packages/arb-token-bridge-ui/next.config.js

```diff
@@ -8,7 +8,9 @@ module.exports = {
 
     return config
   },
-
+  images: {
+    domains: ['www.mexc.com']
```

Comment:
Do we need this?

Reply:
Did some search yesterday while reviewing and [this is what was found ](https://stackoverflow.com/questions/71235874/how-to-allow-all-domains-for-image-nextjs-config)
<img width="667" alt="image" src="https://user-images.githubusercontent.com/7558499/226876763-99dc7380-838e-492e-8119-a638ba41e007.png">.

Another way to escape this would be to host even this image in our repo just like we're doing with the others.

Reply:
Right, but we're not serving any images from that domain.

Reply:
It was actually pulled directly from mexc website before, not from the portal. I saw the warning when I moved to `next/image`